jcl/src/openj9.jvm/share/classes/com/ibm/jvm/Stats.java

```diff
@@ -21,33 +20,49 @@
  *
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
-
 package com.ibm.jvm;
 
 /**
  * This class is used provide the following java heap and system statistics:
- * <ul> committed heap memory
- * <ul> used heap memory
- * <ul> max heap memory
- * <ul> softmx heap memory
- * <ul> free physical memory
- * <ul> total physical memory
- * All this info is retrieved without any allocation of new objects
+ * <ul>
+ * <li>committed heap memory</li>
+ * <li>used heap memory</li>
+ * <li>max heap memory</li>
+ * <li>softmx heap memory</li>
+ * <li>free physical memory</li>
+ * <li>total physical memory</li>
+ * </ul>
+ * All this info is retrieved without any allocation of new objects.
  */
 public class Stats {
-	private long committedHeap = 0L;
-	private long usedHeap = 0L;
-	private long maxHeap = 0L;
-	private long softmxHeap = 0L;
-	private long freePhysicalMem = 0L;
-	private long totPhysicalMem = 0L;
-	private double sysLoadAvg = 0.0;
-	private long cpuTime = 0L;
+	private long committedHeap;
+	private long usedHeap;
+	private long maxHeap;
+	private long softmxHeap;
+	private long freePhysicalMem;
+	private long totPhysicalMem;
+	private double sysLoadAvg;
+	private long cpuTime;
+
+	/**
+	 * Construct a new instance.
+	 */
+	public Stats() {
+		super();
+		this.committedHeap = 0;
```

Comment:
The variables are automatically initialized to zero without adding the bytecodes to explicitly do this. We've removed code like this from other places in the past to (slightly) reduce the code size and improve execution speed.